src/videotile/DefaultVideoTile.ts

```diff
@@ -62,11 +62,10 @@ export default class DefaultVideoTile implements DevicePixelRatioObserver, Video
   }
 
   /**
-   * Disconnect a video stream to a video element by clearing the srcObject of the video element.
-   * This will also stop all the tracks of the current stream in the srcObject.
+   * Disconnect a video stream from a video element by clearing the srcObject of the video element.
```

Comment:
nit: We have used four phrases: `clear`, clear out`, `clean`, and `clean up`. Optionally, we can explicitly state that set `HTMLMediaElement.srcObject` to `null` in some comments.

Reply:
Fixed thanks for the suggestion. Updated elsewhere to use `HTMLVideoElement.srcObject` to `null` over others.